meltdown-mitigation/conditionals.py

```diff
@@ -1,31 +1,28 @@
-""" Meltdown Mitigation exercise """
+from typing import Union
 
 
-def is_criticality_balanced(temperature, neutrons_emitted):
-    """Verify criticality is balanced.
-
-    :param temperature: temperature value in kelvin (integer or float)
-    :param neutrons_emitted: number of neutrons emitted per second (integer or float)
-    :return:  boolean True if conditions met, False if not
```

Comment:
Don't remove default comments like this docstring. I see you wrote your own format, but then there's [Sphinx signature format](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-signatures) which is de-facto a standard for Python docstrings.